java/swagger-petstore/src/main/java/io/swagger/petstore/data/PetData.java

```diff
@@ -0,0 +1,147 @@
+/**
```

Comment:
Either have copyright header in all java files or in none. also cp 2018 not 2016

Reply:
Done!